src/main/webapp/css/welcome.css

```
@@ -0,0 +1,301 @@
+:root {
+    --text-01: #000000;
+    --light-01: #F9F9F9;
+    --light-02: #FFFFFF;
+    --brand-01: #F9F9F9;
+    --brand-02: #f7cd40;
+    --hover-timing: all 0.2s ease;
+    -webkit-font-smoothing: antialiased;
+    --pg-margin: 80px;
+    font-smoothing: antialiased;
+    scroll-behavior: smooth;
+}
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+svg {
+    width: 1.2em;
+    height: 1.2em;
+    stroke: var(--text-01);
+}
+
+svg.bad {
+    stroke: none;
+    color: var(--text-01);
+}
+
+svg #icon-profile.bad {
+    width: 0.5em;
+    height: 0.5em;
+}
+
+a {
+    text-decoration: none;
+    color: var(--text-01);
+}
+
+body {
+    font-family: 'Helvetica Neue', sans-serif;
+    font-weight: 500;
+    transition: var(--hover-timing);
+    background: var(--light-01);
+    width: 100vw;
+    overflow-x: hidden;
+}
+
+header {
+    height: auto;
+    position: sticky;
+    top: 0;
+    border: 1px solid var(--brand-01);
+    z-index: 100;
+}
+
+nav {
+    color: var(--text-01);
+    font-weight: 600;
+    height: 64px;
+    background: var(--light-01);
+    display: flex;
+    flex-direction: row;
+    /*align-items: center;*/
+    /*justify-content: space-between;*/
+    padding: 0 var(--pg-margin);
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+    z-index: 100;
+}
+
+nav ul.navigation-menu {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+    justify-content: center;
+    position: relative;
+    top: 0;
+    z-index: 1;
+    background: none;
+    width: auto;
+    position: relative;
+    align-items: center;
+    padding-left: 12.3em;
+}
+
+nav .navigation-menu a {
+    font-size: 16px;
+    text-decoration: none;
+    color: var(--text-01);
+}
+
+nav .navigation-menu > li {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+nav .navigation-menu > li > a {
+    position: relative;
+    padding: 0 20px;
+    height: 64px;
+    display: flex;
+    align-items: center;
+    justify-items: center;
+    font-weight: 500;
+}
+
+nav .navigation-menu > li > a:hover {
+    font-weight: 700;
+    background: linear-gradient(var(--light-01) 95%, var(--brand-02) 5%);
+}
+
+nav > #logo {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    grid-gap: 8px;
+    line-height: 100%;
+}
+
+nav > #utilities {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    grid-gap: 20px;
+    cursor: pointer;
+}
+
+nav > .search-container{
+    width: 490px;
+    display: block;
+    margin: 0 auto;
+}
+
+input#search-bar{
+    height: 1.2em;
+    /*border: 1px solid #D0CFCE;*/
+    outline: none;
+    border:none;
+    border-bottom: 1px solid var(--text-01);
+    padding: 5px 10px;
+    background: none;
+    /*visibility: hidden;*/
+    /*outline: none;*/
+}
+
+.search-icon{
+    position: relative;
+    float: right;
+}
+
+.profile-menu-wrap {
+    position:absolute;
+    top: 100%;
+    right: 5%;
+    width: 220px;
+    max-height: 0px;
```

```suggestion
    max-height: 0;
```
The unit isn't necessary when the value is 0.